src/jobs/NotificationTrigger/notificationHelper.js

```diff
@@ -0,0 +1,37 @@
+const { NotificationDbService } = require('../../db-service')
+const { getStartEndTime } = require('./notificationTime')
```

Comment:
day start and end time  will be different for different user.

Reply:
@syuraj  sir okay